code/go/pkg/validator/validator.go

```diff
@@ -0,0 +1,21 @@
+package validator
+
+import (
+	"github.com/elastic/package-spec/code/go/internal/validator"
+)
+
+// ValidateFromPath validates a package located at the given path against the
+// appropriate specification and returns any errors.
+func ValidateFromPath(packageRootPath string) error {
```

Comment:
This is the method I was referring in different PR: ValidateFromPath. I would be cool not to load specs every time.

Reply:
Yep, see my reply on the other PR: https://github.com/elastic/package-spec/pull/12#discussion_r467763167. Let's continue the conversation there.